### DISCUSSION TITLE
Project 2: Dicom Support

### DISCUSSION THREAD

**Discussion by CGDogan** [camicroscope/GSOC#43] [General]
Dear @tcpan,

I'm the GSOC Summer intern for this year and working with you, I'll be contributing an efficient DICOM server as I've detailed in my proposal. Could you please provide me some feedback regarding my proposal? By the way, since OpenSlide recently got DICOM support, would you like me to start from reimplementing the openslide [example deepzoom server](https://github.com/openslide/openslide-python/blob/main/examples/deepzoom/deepzoom_server.py) in C?

**Comment by tcpan** [comment]
@CGDogan, - congratulations and welcome to the project!  Let's schedule a time to video conference this week  what's your availability like?  
Yes, starting from existing openslide is a good idea.  The tile serving backend hasn't been touched in several years, so there is likely some work involved in updating the openslide version and integration testing.  

Thanks!

**Comment by CGDogan** [comment]
@tcpan All times in UTC, today I'm available from 1700 to 2200, on wednesday from 1300 to 2200, on thursday from 1700 again.

Yes, some interesting subtasks would be running the OpenSlide wrapper and also the fork of iipsrv for openslide (https://github.com/cytomine/iipsrv, can be built wit https://github.com/cytomine/Dockerfiles/blob/master/iipCyto/Dockerfile) and making it work for dicom.

**Comment by tcpan** [comment]
how about 9 EDT Wed, I think that's 1300 UTC...

Thanks!  Tony

**Comment by CGDogan** [comment]
9 EDT WED, 13 UTC. thanks. Are you able to set up a Zoom meeting or we can
use WhatsApp video calls on desktop?

**Comment by tcpan** [comment]
I’ll send you a MS Teams meeting invite. Thanks!Tony From: CGDoganSent: Tuesday, May 9, 2023 11:10 AMTo: camicroscope/GSOCCc: Tony Pan; MentionSubject: Re: [camicroscope/GSOC] Project 2: Dicom Support (Discussion #43) 9 EDT WED, 13 UTC. thanks. Are you able to set up a Zoom meeting or we canuse WhatsApp video calls on desktop?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by tcpan** [comment]
BTW, I am not sure why t’s using my personal gmail account.  Please use ***@***.*** What is your email address? Thanks!  From: CGDoganSent: Tuesday, May 9, 2023 11:10 AMTo: camicroscope/GSOCCc: Tony Pan; MentionSubject: Re: [camicroscope/GSOC] Project 2: Dicom Support (Discussion #43) 9 EDT WED, 13 UTC. thanks. Are you able to set up a Zoom meeting or we canuse WhatsApp video calls on desktop?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by CGDogan** [comment]
My email address is c g d 30 At cam ac uk. If notifications are coming to
the wrong address, you might need to update it from GitHub settings for
your username tcpan.

**Comment by CGDogan** [comment]
@tcpan  I haven't received a Teams invitation yet, is everything alright?

**Comment by tcpan** [comment]
Good morning.  It’s coming.  My password expired and need to fix it first.  I might be a few minutes late in sending this.  Sorry about that.  From: CGDoganSent: Tuesday, May 9, 2023 11:40 AMTo: camicroscope/GSOCCc: Tony Pan; MentionSubject: Re: [camicroscope/GSOC] Project 2: Dicom Support (Discussion #43) My email address is c g d 30 At cam ac uk. If notifications are coming tothe wrong address, you might need to update it from GitHub settings foryour username tcpan.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by tcpan** [comment]
Let’s use google meets for now:  To join the video meeting, click this link: https://meet.google.com/bcd-svbj-qwfOtherwise, to join by phone, dial +1 224-901-2385 and enter this PIN: 756 158 387# Tony From: TonySent: Wednesday, May 10, 2023 8:54 AMTo: camicroscope/GSOCSubject: RE: [camicroscope/GSOC] Project 2: Dicom Support (Discussion #43) Good morning.  It’s coming.  My password expired and need to fix it first.  I might be a few minutes late in sending this.  Sorry about that.  From: CGDoganSent: Tuesday, May 9, 2023 11:40 AMTo: camicroscope/GSOCCc: Tony Pan; MentionSubject: Re: [camicroscope/GSOC] Project 2: Dicom Support (Discussion #43) My email address is c g d 30 At cam ac uk. If notifications are coming tothe wrong address, you might need to update it from GitHub settings foryour username tcpan.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
